src/components/NFTBook/CollectionItemCard.vue

```diff
@@ -0,0 +1,147 @@
+          >
+            <NFTBookCoverWithFrame
+              :class="['w-full', 'rounded-t-[inherit] rounded-b-[0]']"
+              :src="imageSrc"
+              :alt="collectionName"
+              :cover-resize="450"
+              class-aspect-ratio="aspect-[1]"
+            />
+          </div>
+          <div
+            :class="[
+              'flex',
+              'flex-col',
+              'text-center',
+              'whitespace-pre-line',
+              'px-[24px]',
+              'pt-[48px]',
+              'py-[24px]',
+              'bg-white',
+              'relative',
+              'rounded-b-[inherit] !rounded-t-[0]',
+            ]"
+          >
+            <div class="flex flex-col items-center justify-center mt-[-70px]">
+              <Identity
+                :avatar-url="ownerAvatar"
+                :avatar-size="40"
+                :is-lazy-loaded="true"
+              />
+              <Label
+                class="w-full mt-[8px] text-like-green font-[600]"
+                content-class="min-w-0"
+                align="center"
+              >
+                <span class="text-medium-gray">by</span>&nbsp;
+                <span class="truncate">{{ ownerDisplayName }}</span>
+              </Label>
+            </div>
+            <Label preset="h5" class="mt-[12px] break-normal" align="center">{{
+              collectionName
+            }}</Label>
+            <div class="flex items-center justify-center mt-[16px]">
+              <ButtonV2
+                preset="secondary"
+                :text="priceLabel"
+                :is-disabled="!stock"
+                @click="handleClickCollect"
+              >
+                <template #prepend>
+                  <NFTWidgetIconInsertCoin class="w-[16px]" />
+                </template>
+              </ButtonV2>
+            </div>
+          </div>
+        </NFTPortfolioCard>
+      </template>
+    </NFTGemWrapper>
+  </NuxtLink>
+</template>
+<script>
+import { mapActions, mapGetters } from 'vuex';
+import { parseNFTMetadataURL } from '~/util/nft';
+import collectionMixin from '~/mixins/nft-collection';
+
+export default {
+  name: 'NFTBookCollectionItemCard',
+  mixins: [collectionMixin],
+  props: {
+    collectionId: {
+      type: String,
+      default: '',
+    },
+  },
+  data() {
+    return {
+      data: null,
+    };
+  },
+  computed: {
+    ...mapGetters([
+      'getNFTCollectionInfoByCollectionId',
+      'getUserInfoByAddress',
+    ]),
+    detailsPageRoute() {
+      return this.localeLocation({
+        name: 'nft-collection-collectionId',
+        params: { collectionId: this.collectionId },
+      });
+    },
+    priceLabel() {
+      return this.formattedCollection?.priceLabel;
+    },
+    stock() {
+      return this.formattedCollection?.stock;
+    },
+    imageSrc() {
+      return parseNFTMetadataURL(this.formattedCollection?.image);
+    },
```

Comment:
Can we use `this.collectionImageUrl` in mixin?

Reply:
[♻️ Use collection mixin data instead of new computed props](https://github.com/likecoin/liker-land/pull/1863/commits/bb0ace8d14e892f348a3b53982b387a41cbc4fde)